x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/SequenceMatcher.java

```diff
@@ -207,14 +206,19 @@ public boolean hasCandidates(int stage) {
         return false;
     }
 
-    public Payload payload(long startTime) {
-        TimeValue tookTime = new TimeValue(System.currentTimeMillis() - startTime);
-        List<Sequence> view = limit != null ? limit.view(completed) : completed;
-        Payload p = new SequencePayload(view, false, tookTime);
-        clear();
-        return p;
+
+    public List<Sequence> completed() {
+        return limit != null ? limit.view(completed) : completed;
     }
 
+    //    public Payload payload(long startTime) {
+    //        TimeValue tookTime = new TimeValue(System.currentTimeMillis() - startTime);
+    //
+    //        Payload p = new SequencePayload(view, false, tookTime);
+    //        clear();
+    //        return p;
+    //    }
```

Comment:
Need to remove this as well.

Reply:
`// TESTRESPONSE[skip: response format changed]` should work here too. I can fix this as part of a follow-up PR.